pages/quantum.js

```diff
@@ -53,139 +61,453 @@ import {
+            <h4 class="TextFormattingH4">ControlledDialog</h4>
+            <h6 class="TextFormattingH6">
+              ControlledDialog is a button that activates a dialog window in
+              front of all content. It is commonly used to provide an important
+              information or ask for some user interaciont.
+            </h6>
+            <ControlledDialog />
+          </div>
+          <div class="PaddingTopBotttom ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">Hamburger</h4>
+            <h6 class="TextFormattingH6">
+              Hamburger component is used for opening mobile menu. You can also
+              pass a aria-label description to inform screen readers that there
+              are new notifications.
+            </h6>
+            <Hamburger inverted />
+          </div>
+          <div class="PaddingTopBotttom ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">SegmentedControl</h4>
+            <h6 class="TextFormattingH6">
+              The Segmented control component is a linear set of two or more
+              alternation buttons, with all segments with same width.
+            </h6>
+            <SegmentedControl name="items-text" items={segmentedControlMock} />
+          </div>
+          <div class="PaddingTopBotttom ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">ControlledModal</h4>
+            <h6 class="TextFormattingH6">
+              Modals inform users about a task and can contain critical
+              information, require decisions, or involve multiple tasks. A
+              ControlledModal contains control buttons, such as "Cancel" and
+              "Delete".
+            </h6>
+            <ControlledModal />
+          </div>
+          <div class="PaddingTopBotttom ComponentsContainer WhiteBackground RoundedEdges">
+            <h4 class="TextFormattingH4">ControlledSnackBar</h4>
+            <h6 class="TextFormattingH6">
+              SnackBars are used to show an alert about the action that have
+              just been made: delete or archive content by accident (undo),
+              resend messages that not been sent for some reason, or try to
+              reconnect to internet (retry). A ControlledSnackBar has control
+              buttons.
+            </h6>
+            <ControlledSnackBar />
+          </div>
+        </div>
+      </div>
+      <div class="FlexContainer ColoredBackground ComponentsContainer">
+        <div class="Width">
+          <div class=" WhiteBackground  RoundedEdges ComponentsContainer">
+            <h4 class="TextFormattingH4">Badge</h4>
+            <h6 class="TextFormattingH6">
+              Badges are used for showing the quantity of something, such as
+              warnings, inbox messages and others.
+            </h6>
+            <Badge>Example Badge</Badge>
```

Comment:
![image](https://github.com/catho/ssr-quantum-simulator/assets/3389749/3f1cacd0-6a2f-4f61-b3c9-d31d7236331e)

Reply:
I've inserted a Tag inside the Badge and solved the issue